packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Unreleased
+- [feature] Added `firebase.firestore.FieldValue.arrayUnion()` and
+  `firebase.firestore.FieldValue.arrayRemove()` to atomically add and remove
+  elements from an array field in a document.
+- [feature] Added `'array-contains'` query operator for use with `.where()` to
```

Comment:
Is the double-quoting intentional for better formatting?

Reply:
yep